docs/conveyor/user-guide.md

```diff
@@ -0,0 +1,49 @@
+# Getting started
+## Preparation
+At first, you need to get some DAI and USDC tokens, the following steps show how to get these tokens on the Ethereum Ropsten testnet
+
+1. using [ropsten faucet](https://faucet.ropsten.be/) to get eth of ropsten testnet
+2. using [uniswap](https://app.uniswap.org/#/swap) to swap DAI and USDC, don't forget to connect to the ropsten test network in your Metamask
```

Comment:
2. Swap ETH for USDC and DAI tokens on [uniswap](https://app.uniswap.org/#/swap). Don't forget to connect Metamask to the ropsten test network.